cmake/shared/PACE_CodeAnalysis.cmake

```diff
@@ -2,10 +2,14 @@ add_custom_target(analyse
   COMMENT "Performing code analysis..."
   )
 
+# Handle CMAKE list to matlab cell array
+string(JOIN "','" IGNORE_STRING ${PACE_MLINT_IGNORE})
+string(CONCAT IGNORE_STRING "{'" IGNORE_STRING "'}")
+
 add_custom_target(analyse-mlint
   COMMENT "- Performing MATLAB analysis (Mlint)..."
   BYPRODUCTS "${CMAKE_CURRENT_BINARY_DIR}/mlint.out"
-  COMMAND ${Matlab_MAIN_PROGRAM} -nodisplay -batch "\"addpath('${Herbert_ROOT}/admin');lint_json({'${CMAKE_SOURCE_DIR}/**/*.m'},'${CMAKE_CURRENT_BINARY_DIR}/mlint.json');exit\""
+  COMMAND ${Matlab_MAIN_PROGRAM} -nodisplay -batch "\"addpath('${Herbert_ROOT}/admin');lint_json({'${CMAKE_SOURCE_DIR}/**/*.m'},'${CMAKE_CURRENT_BINARY_DIR}/mlint.json','exclude',${IGNORE_STRING});exit\""
```

Comment:
As the command string is now getting quite long ad complex would it be clear to build it as a variable outside the `add_custom_target` call?

For readability can this be split over multiple lines too?